modules/doc/training/content/sqa/training/lead/sqa_release.md

```diff
@@ -0,0 +1,152 @@
+# Performing MOOSE and MOOSE-based Application Release(s)
+
+!---
+
+## PLN-4005 and Releases
+
+!alert warning title=Refer to PLN-4005
+PLN-4005 details what must be performed during a release of MOOSE and MOOSE-based applications, it
+should be utilized as a reference. This presentation is an introduction to aid in following the
+procedure within PLN-4005.
+
+!---
+
+## Overview of Release Process
+
+1. Select a stable (main/master branch) for release
+1. Download testing results
+1. Complete second review
+1. Document the review by creating "release log"
+1. Update MooseDocs config.yml to point to downloaded results
+1. Commit the results and log
+1. Use `git tag` to label the release
+1. Push the tag to the main repository
+
+!---
+
+## Select Revision for Release
+
+Any revision of MOOSE or MOOSE-based application can be selected for a release as long as all
+the testing is complete and passed.
+
+!alert note title=Releases +must+ depend on releases
+A MOOSE-based application release must use release versions of MOOSE and dependent
+applications. When releasing an application a Merge/Pull Request should be created with the
+MOOSE and application submodules updated to the release tags. The following process is then followed
+after merging and testing of this change is complete.
+
+!---
+
+## Download Results
+
+Checkout the branch for release and execute the `git log` command.
+
+```
+$ git log
+commit b657ae26aae14fbc91ad2651a364cfad80c2b9ac (HEAD -> master, upstream/master)
+Merge: b107961dbd 4812111857
+Author: moosetest <bounces@inl.gov>
+Date:   Wed Sep 29 11:22:23 2021 -0600
+
+    Merge commit '4812111857205877df7bb943958f9cd1f32b0d27'
+```
+
+The hashes in this message can be used to obtain results from CIVET, as shown in the links below.
+The downloaded tag.gz files should be committed in a "release" directory at the top level of the
+repository. The files should be renamed to follow the pattern "results_12345.tar.gz", where the
+number is the job number.
```

Comment:
For future training clarification, perhaps "job number" should be described in a little more detail?

Reply:
I am going to remove the need to rename the files